resources/zib/zib-Vaccination-event.xml

```diff
@@ -95,12 +95,11 @@
     <element id="Immunization.doseQuantity">
       <path value="Immunization.doseQuantity" />
       <short value="Dose" />
-      <definition value="The amount of product administered shown in milliliters. In most cases, the entire product is administered; in some cases, a described part of the product is administered." />
+      <definition value="The amount of product administered shown UCUM, often in mL. In most cases, the entire product is administered; in some cases, a described part of the product is administered." />
+      <comment value="The context of use may frequently define what kind of quantity this is and therefore what kind of units can be used. The context of use may also restrict the values for the comparator.&#xD;&#xA;&#xD;&#xA;The definition of Dose deviates slightly from the zib 2020 vaccination definition; The unit of measure is not limit to mL. This difference is according to the planned vaccination zib modifications (https://bits.nictiz.nl/browse/ZIB-1654)." />

```

Comment:
@pieter-edelman-nictiz I added a comment of the deviating Dose definition. Is it formulated correctly?

Reply:
Just overwrite the existing comment.

Reply:
As has been done?